cypress/integration/inventory/update-effective-location-for-item.spec.js

```diff
@@ -9,14 +9,14 @@ import getRandomPostfix from '../../support/utils/stringTools';
 import generateItemBarcode from '../../support/utils/generateItemBarcode';
 import permissions from '../../support/dictionary/permissions';
 import Users from '../../support/fragments/users/users';
-import InventoryHoldings from '../../support/fragments/inventory/holdings/inventoryHoldings';
+import InventoryInstances from '../../support/fragments/inventory/inventoryInstances';
 
-describe('Update the effective location for the item', () => {
+describe('ui-inventory: Update the effective location for the item', () => {
   const instanceTitle = `autoTestInstanceTitle.${getRandomPostfix()}`;
   const anotherPermanentLocation = 'Main Library';
   const ITEM_BARCODE = generateItemBarcode();
   let userId = '';
```

Comment:
Do you need to initialized this variable userId?

Reply:
yes, I delete user by user id

Reply:
I understand that you need this variable. Do you need give this variable a value equals to ''? I guess no because you don't use this value anywhere. Later you can spend more time to understand the possible error because of the initialization of the variable like in this example, but if you notice, for example undefined value, you will realize that the value is not passed to this variable.

Reply:
Done